### DISCUSSION TITLE
Could not find InternSet in d3 with Webpack

### DISCUSSION THREAD

**Discussion by nmondon** [observablehq/plot#400] [Q&A]
Hi and thanks for the nice work here!

I've already try and love Plot in observablehq. 
But when I try to integrate it in my website built with Webpack, I get an error `new d3.InternSet is not a constructor`. 

I'm using d3@6.7 and the latest @observablehq/plot (0.1.0).

**Comment by mbostock** [comment]
Hi there. I’ve converted this to a discussion because this isn’t a Webpack support forum, but I’ll leave this open in case anyone else visiting has tips on using Webpack. Good luck.
